discover/portal/articles/100-web-experience-management/02-building-a-site/03-creating-multiple-navigation-menus/00-creating-and-managing-navigation-menus-intro.markdown

```diff
@@ -1,22 +1,22 @@
 # Managing Site Navigation [](id=managing-site-navigation)
 
-With @product-ver@, pages can be decoupled from menus. Navigation Menus let you
-arrange your page taxonomy any way you want. Marketers can leave one-off
-marketing landing pages out of the navigation, and you can create 
-multiple navigation menus: a main menu, secondary menus, footer menus, and
-custom menus for anything that you can dream up.
+Liferay provides straightforward but powerful tools for creating and organizing 
+pages. You can create pages in a simple, flat site navigation, or you can 
+create sub-pages and build the page hierarchy any way you like.
+
+With @product-ver@, you can create Navigation Menus that are separate from your 
+primary navigation. Navigation Menus let you arrange pages any way you want. 
```

Comment:
This sentence is using "primary navigation" to refer to using the Page Hierarchy as a Menu. That is incorrect since we also have a concept of a menu that is acting as a primary navigation. It could be used, for example, in sites where it's desired to use a Navigation Menu as the primary navigation and not the Page Hierarchy.